src/Admin/DevToolsUserAccess.php

```diff
@@ -72,11 +75,24 @@ public function get_user_enabled( $user ) {
 		if ( ! $user instanceof WP_User ) {
 			$user = new WP_User( $user );
 		}
-		$value = $user->get( self::USER_FIELD_DEVELOPER_TOOLS_ENABLED );
-		if ( '' === $value ) {
-			$value = true; // @todo Allow to be filtered so that the default status can be changed.
+		$enabled = $user->get( self::USER_FIELD_DEVELOPER_TOOLS_ENABLED );
+		if ( '' === $enabled ) {
+			// Disable Developer Tools by default when in Reader mode.
+			$enabled = AMP_Theme_Support::READER_MODE_SLUG !== AMP_Options_Manager::get_option( Option::THEME_SUPPORT );
+
+			/**
+			 * Filters whether Developer Tools is enabled by default for a user.
+			 *
+			 * When Reader mode is active, Developer Tools is currently disabled by default.
+			 *
+			 * @since 2.0.1
+			 *
+			 * @param bool $enabled DevTools enabled.
+			 * @param int  $user_id User ID.
```

Comment:
Not a blocker but would we need a `@return` annotation for documentation purposes?

Reply:
We don't use `@return` for other filter docs, do we? Core doesn't either. The type of the first arg is implied to be the return type.

Reply:
Example:

https://github.com/ampproject/amp-wp/blob/8d0409134f4c509e317a40bb815371f8d67e4e3e/includes/class-amp-post-type-support.php#L182-L190

Reply:
One example:

https://github.com/ampproject/amp-wp/blob/8d0409134f4c509e317a40bb815371f8d67e4e3e/includes/class-amp-theme-support.php#L2272-L2280

But yea, the return type will can be inferred from the first argument in most cases.

Reply:
See [WordPress inline documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters:~:text=Note%20that%20%40return%20is%20not%20used,hooks%20always%20return%20their%20first%20parameter.):

> ![image](https://user-images.githubusercontent.com/134745/92179985-c4591200-edfa-11ea-8537-b0172d423178.png)

Reply:
Ah so those instances should be removed then.